Documentation/helm-operator.md

```diff
@@ -94,12 +94,13 @@ The following tables lists the configurable parameters of the rook-operator char
 | `rbacEnable`       | If true, create & use RBAC resources | `true`               |
 | `resources`        | Pod resource requests & limits       | `{}`                 |
 | `logLevel`         | Global log level        | `INFO`                 |
-| `agent.flexVolumeDirPath` | Path where the Rook agent discovers the flex volume plugins | `/usr/libexec/kubernetes/kubelet-plugins/volume/exec/` |
+| `agent.flexVolumeDirPath` | Path where the Rook agent discovers the flex volume plugins (*) | `/usr/libexec/kubernetes/kubelet-plugins/volume/exec/` |
 | `agent.toleration`        | Toleration for the agent pods | <none> |
 | `agent.tolerationKey`     | The specific key of the taint to tolerate | <none> |
 | `mon.healthCheckInterval` | The frequency for the operator to check the mon health | `45s` |
 | `mon.monOutTimeout`       | The time to wait before failing over an unhealthy mon | `300s` |
 
+&ast; - For Kubernetes 1.9.x `agent.flexVolumeDirPath` should be changed to `/var/lib/kubelet/volumeplugins/`. [Flexvolume documentation](https://github.com/rook/rook/blob/master/Documentation/flexvolume.md#for-kubernetes--19x)
```

Comment:
just a minor suggestion, to remove the `-`. With the asterisk it looks odd. If you could make that minor update and squash back to one commit, it'll be ready to merge. Thanks!

Reply:
one more thing... you can actually just use a relative path to other documentation. Both github and the rook.io docs will then link to the correct version of the doc. So this would do in the link: 
```
flexvolume.md#for-kubernetes--19x
```